downstream/modules/platform/ref-controller-cleanup-expired-tokens.adoc

```diff
@@ -0,0 +1,15 @@
+[id="ref-controller-cleanup-expired-tokens"]
+
+= Cleanup Expired OAuth2 Tokens
+
+To remove expired OAuth2 tokens, click on the launch image:rightrocket.png[Launch,15,15] icon next to *Cleanup Expired OAuth2 Tokens*.
+
+You can review or set a schedule for cleaning up expired OAuth2 tokens by performing the same procedure described for activity stream
+management jobs. 
+
+For more information, see xref:proc-controller-scheduling-deletion[Scheduling deletion].
+
+You can also set or review notifications associated with this management job the same way as described in xref:proc-controller-management-notifications[setting notifications] for activity
+stream management jobs.
+
+For more informaation, see [Notifications] in the _{ControllerUG}_.
```

Comment:
see link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/automation_controller_user_guide/controller-notifications[Notifications] in the _{ControllerUG}_.